praw/models/mod_note.py

```diff
@@ -0,0 +1,42 @@
+"""Provide the ModAction class."""
+from typing import TYPE_CHECKING, Union
+
+from .base import PRAWBase
+
+if TYPE_CHECKING:  # pragma: no cover
+    import praw
+
+
+class ModNote(PRAWBase):
+    """Represent a moderator action."""
+
+    def __str__(self) -> str:
+        """Return a string representation of the instance."""
+        return getattr(self, "id")
+
+    @property
+    def user(self) -> "praw.models.Redditor":
+        """Return the :class:`.Redditor` who the action was issued by."""
+        return self._reddit.redditor(self._user)  # pylint: disable=no-member
+
+    @user.setter
+    def user(self, value: Union[str, "praw.models.Redditor"]):
+        self._user = value  # pylint: disable=attribute-defined-outside-init
+
+    @property
+    def operator(self) -> "praw.models.Redditor":
```

Comment:
The property in the returned object is literally "operator". Should we name the function mod anyway?

Reply:
I think it would be fine to name it moderator. `operator` is a really odd choice to name the attribute. We could create an alias for `moderator` that points to `operator` instead.

Reply:
Added this as moderator, so the `operator` field will stay as just a string in the object rather than a redditor object